core/src/main/java/bio/terra/pearl/core/dao/portal/PortalEnvironmentDao.java

```diff
@@ -38,7 +38,8 @@ public Class<PortalEnvironment> getClazz() {
     }
 
     public List<PortalEnvironment> findByPortal(UUID portalId) {
```

Comment:
please add a comment clarifying that this returns them with the configs attached